src/platform/cocoamain.mm

```diff
@@ -1218,6 +1218,227 @@ - (IBAction)preferences:(id)sender {
     [NSApp stop:nil];
 }
 
+#ifdef HAVE_SPACEWARE
```

Comment:
I mean, if you do not set `SPACEWARE_FOUND` to true then that's not needed. (And the library is indeed not found at compile-time.)